.github/actions/run-smoke-tests/action.yml

```diff
@@ -0,0 +1,66 @@
+name: Run Smoke Tests
+description: Executes Smoke Tests
+inputs:
+  role-to-assume:
+    description: an IAM role to use in tests
+    required: true
+  region:
+    description: an AWS region to run in
+    required: false
+    default: us-west-2
+runs:
+  using: 'composite'
+  steps:
+    - name: Verify Amplify Path
+      shell: bash
+      run: |
+        if [[ -z "$AMPLIFY_PATH" ]]; then
+          echo "AMPLIFY_PATH must be defined"
+          exit 1
+        fi
+        echo "AMPLIFY_PATH=$AMPLIFY_PATH"
+        $AMPLIFY_PATH version
+
+    - name: Install Dependencies
+      shell: bash
+      run: yarn install
```

Comment:
should this include `--frozen-lockfile`?

Reply:
```
bcd0740fa72c:amplify-cli-nofork sobkamil$ yarn install --frozen-lockfile
➤ YN0050: The --frozen-lockfile option is deprecated; use --immutable and/or --immutable-cache instead
```

I'll add immutable.